scripts/common/kubernetes.sh

```diff
@@ -1,19 +1,18 @@
-export KUBECTL_PLUGINS_PATH=/usr/local/bin
 
 function kubernetes_host() {
     kubernetes_load_ipvs_modules
 
     kubernetes_sysctl_config
 
+    kubernetes_install_host_packages "$KUBERNETES_VERSION"
```

Comment:
Why was this moved to before `kubernetes_get_host_packages_online` ?

Reply:
kubernetes_install_host_packages calls kubernetes_get_host_packages_online at the top of the function. kubernetes_install_host_packages is called in other places so im trying to avoid making changes to it.

Reply:
In that case the block beginning on 12 should be removed. That will never be true.

Reply:
it will be if `kubernetes_host_commands_ok`

```
# k8sVersion is an argument because this may be used to install step versions of K8s during an upgrade
# to the target version
function kubernetes_install_host_packages() {
    k8sVersion=$1
    logStep "Install kubelet, kubeadm, kubectl and cni host packages"
    if kubernetes_host_commands_ok "$k8sVersion"; then
        logSuccess "Kubernetes host packages already installed"
        return
    fi
    if [ "$AIRGAP" != "1" ] && [ -n "$DIST_URL" ]; then
        kubernetes_get_host_packages_online "$k8sVersion"
    fi
```

Reply:
And that's needed to get the images that are loaded right after